var/spack/repos/builtin/packages/py-setuptools-scm/package.py

```diff
@@ -42,3 +42,5 @@ class PySetuptoolsScm(PythonPackage):
     depends_on("py-tomli@1:", when="@7.1: ^python@:3.10", type=("build", "run"))
     depends_on("py-typing-extensions", when="@7:", type=("build", "run"))
     depends_on("py-importlib-metadata", when="@7: ^python@:3.7", type=("build", "run"))
+
+    depends_on("git", type=("build", "run"))
```

Comment:
Incidentally, this causes `git` to be built as part of bootstrapping dev dependencies